frontend/client/__tests__/__snapshots__/addmembermodal.react.test.js.snap

```diff
@@ -0,0 +1,120 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Has this been checked for accuracy? I believe Serena created this, do we know that it works properly?

Reply:
I left this file alone in this PR and updated it the test Manage Members PR!